### ISSUE TITLE
Validation loss starts to rise

### ISSUE THREAD

**Issue by YinJerry** [deepgram/kur#27]
I have trained the deepspeech model for about 48 hours and I found the loss is gradually going up. I am using 100 hour train data and have run 20 epochs. Have I got enough training and can stop training? Does this program include language model such as n-gram? If not, I probably need not expect the perfect results. Thanks!
Here are the last two results:
[INFO 2017-03-02 18:11:38,332 kur.model.executor:175] Validation loss: 155.686
Prediction: "i know wat mammock ind a forto guervev en iwile set she give tat"
Truth: "i know what mamma can afford to give and i will see she gives it"
[INFO 2017-03-02 20:43:30,224 kur.model.executor:175] Validation loss: 159.761
Prediction: "i se at call tad his vholly sendy oll o oadanetural ho those obof"
Truth: "i see him called tad his voice sounding hollow and unnatural to those above"

**Comment by scottstephenson** [comment]
Your results are good! There isn't a language model included so that's pretty much the output you can expect with that dataset.

**Comment by YinJerry** [comment]
Aha, I see. Thank you for your support! It's fun to try your great work! Do you have any further plan to develop next version of deepspeech? Probably I can add a language model to make it better when I have time. :)
